src/Spryker/Zed/CodeGenerator/Business/CodeGeneratorFacadeInterface.php

```diff
@@ -10,6 +10,9 @@
 interface CodeGeneratorFacadeInterface
 {
     /**
+     * Specification:
+     * - Create Zed bundle CodeGenerator.
```

Comment:
`Creates ...`.
All the specifications must start from a verb in present simple 3rd person. Please adjust everywhere.